src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/First.txt

```diff
@@ -69,3 +69,22 @@ Blank()
 
 >> FirstN(Filter([1,2,3],Value=4),2)
 Table()
+
+// ********** Coercions ************
+>> FirstN([1,2,3,4,5], "3")
+Table({Value:1},{Value:2},{Value:3})
+
+>> LastN([1,2,3,4,5], "3")
+Table({Value:3},{Value:4},{Value:5})
+
+>> FirstN([1,2,3,4,5], true)
+Table({Value:1})
+
+>> LastN([1,2,3,4,5], true)
+Table({Value:5})
+
+>> FirstN([1,2,3,4,5], Date(1900,1,1))
```

Comment:
I applaud the test , but this is so painful to see - I can't imagine a case where the customer ever intends to write this. Is there some way we can error or at least warn here? 
.  It's like #984.

Reply:
Good point, this is indeed bizarre (albeit possible). I'll remove those tests, since we don't want to promote this usage. And we should have an internal discussion to decide what to do with it. On one hand, we want to have parity with Excel, and it does support that. On the other hand, it really doesn't make sense, so a warning would be a good compromise.